extensions/panache/mongodb-panache/runtime/src/main/java/io/quarkus/mongodb/panache/transaction/interceptor/ClienSessionHolder.java

```diff
@@ -0,0 +1,24 @@
+package io.quarkus.mongodb.panache.transaction.interceptor;
+
+import java.util.Optional;
+
+import com.mongodb.client.ClientSession;
+
+public final class ClienSessionHolder {
```

Comment:
I don't think you need this class. And its name has a missing `t` at `ClientT`.

Reply:
Added the missing `t` will remove it if I achieve to retrieve the session from the transaction manager.